db/1.template.sql

```diff
@@ -2326,9 +2326,40 @@ CREATE ALGORITHM=UNDEFINED SQL SECURITY INVOKER VIEW `UserListing` AS select `Ra
 
 -- 2022-11-27 11:58:31
 
+/* 2022-11-27 add clickable alerts */
+ALTER TABLE `Alerts`
+ADD `Link` text COLLATE 'utf8mb4_bin' NOT NULL;
 
+/* 2022-12-10 add changelog */
+DROP TABLE IF EXISTS `ChangelogEntries`;
+CREATE TABLE `ChangelogEntries` (
+  `Id` int(11) NOT NULL AUTO_INCREMENT,
+  `ChangelogId` int(11) NOT NULL,
+  `Name` text COLLATE utf8mb4_bin NOT NULL,
+  `Tags` text COLLATE utf8mb4_bin NOT NULL,
```

Comment:
Since the columns in the example are clearly JSON arrays, i suggest to put the right "JSON" type to this column.